doc/testing.txt

```diff
@@ -0,0 +1,55 @@
+.. _testing:
```

Comment:
That is true! I forgot to remove the file. We were thinking of only having the testing section in the troubleshooting page under "How do I test if everything works properly?"